doc/api/child_process.markdown

```diff
@@ -261,8 +258,16 @@ The `sendHandle` option to `child.send()` is for sending a TCP server or
 socket object to another process. The child will receive the object as its
 second argument to the `message` event.
 
-Emits an `'error'` event if the message cannot be sent, for example because
-the child process has already exited.
+The `callback` option is a function that is invoked after the message is
+sent but before the target may have received it.  It is called with a single
+argument: `null` on success, or an `Error` object on failure.
+
+`child.send()` emits an `'error'` event if no callback was given and the message
+cannot be sent, for example because the child process has already exited.
```

Comment:
This impacts the cluster API and process API, which surface `.send()`.

I assume process will now have an error event which can be emitted if send fails, and node will crash if it isn't listened on?

For users of cluster, if they fail to listen on `worker.process.on('error', ...`, the master will crash? Or will you re-emit from the process object onto the worker object?